onepassword/data_source_onepassword_item.go

```diff
@@ -218,7 +229,7 @@ func dataSourceOnepasswordItemRead(ctx context.Context, data *schema.ResourceDat
 			data.Set("note_value", f.Value)
 		default:
 			if f.Section == nil {
-				data.Set(strings.ToLower(f.Label), f.Value)
+				data.Set(strings.ReplaceAll(strings.ToLower(f.Label), " ", "_"), f.Value)
```

Comment:
Honestly, this should be `f.ID` instead of `f.Label` as you can have duplicate labels. But this would be a backwards incompatible change, so I didn't change it.